Chess-FromZeroToHero.DataAccess/Repositories/UserRepository.cs

```diff
@@ -0,0 +1,115 @@
+﻿using Chess_FromZeroToHero.Contracts.Dtos.User;
+using Chess_FromZeroToHero.Contracts.Helpers;
+using Chess_FromZeroToHero.DataAccess.Entities;
+using Chess_FromZeroToHero.DataAccess.Pagination;
+using Chess_FromZeroToHero.DataAccess.Repositories.Interfaces;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Identity.Client;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection.Metadata.Ecma335;
+using System.Text;
+using System.Threading.Tasks;
+using System.Xml;
+
+namespace Chess_FromZeroToHero.DataAccess.Repositories
+{
+    public class UserRepository : IUserRepository
+    {
+        private readonly ChessDbContext _dbContext;
+
+        public UserRepository(ChessDbContext dbContext)
+        {
+            _dbContext = dbContext;
+        }
+
+        private async Task<User> GetEntityByIdAsync(Guid id)
```

Comment:
Is this method used? If so it is a duplicate of `GetByIdAsync(Guid id)` and should not return an entity.

Reply:
Yes, it's used in GetByIdAsync and UpdateAsync, but I'll remove it since now it appears to be useless